### PR TITLE
Tests for AO3-3922 co-authoring

### PR DESCRIPTION
**Pull request by cesy**
https://otwarchive.atlassian.net/browse/AO3-3922

### CODE REVIEW

File: features/works/work_edit.feature [hunk 1]
```diff
@@ -131,4 +131,33 @@ Feature: Edit Works
       And I post the work "Load of Typos"
     When I edit the work "Load of Typos"
       And I press "Preview"
-    Then I should not see "draft"
\ No newline at end of file
+    Then I should not see "draft"
+
+  Scenario: You can add a co-author to an already-posted work
```
Comment (sarken) [inline features/works/work_edit.feature]: We have two steps that might be useful in this test:

```
When /^I add the co-author "([^\"]*)" to the work "([^\"]*)"$/ do |coauthor, work|
  step %{I edit the work "#{work}"}
  step %{I add the co-author "#{coauthor}"}
  step %{I post the work without preview}
end

When /^I add the co-author "([^\"]*)"$/ do |coauthor|
  step %{the user "#{coauthor}" exists and is activated}
  check("Add co-authors?")
  fill_in("pseud_byline", :with => "#{coauthor}")
end
```

**Comment by sarken** [comment]
Hi, Cesy! 

For future pull requests, could you please include a link to the JIRA issue as well as including the issue number? It makes reviewing go just a tiny bit quicker, and every little bit helps.

Thanks!

**Comment by cesy** [comment]
Sure, done. I hadn't spotted that it didn't link automatically :s

File: features/works/work_edit.feature [hunk 2]
```diff
@@ -131,4 +131,28 @@ Feature: Edit Works
...
+    Given I am logged in as "leadauthor"
+      And I post the work "Dialogue"
+    When I add the coauthor "coauthor" to the work "Dialogue"
```
Comment (sarken) [inline features/works/work_edit.feature]: It looks like one of the test failures is happening because there's a hyphen missing in "add the co-author"

**Pull request merged**